code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -136,7 +136,7 @@
 	burst_size = 2
 
 /obj/item/weapon/gun/projectile/automatic/l6_saw
-	name = "\improper L6 SAW LMG"
+	name = "\improper L6 SAW"

```

Comment:
SAW is the type of gun though. Your name was "L6 squad automatic support weapon light machine gun"